gst/isomp4/qtdemux.c

```diff
@@ -7042,14 +6931,7 @@ qtdemux_parse_cenc (GstQTDemux * qtdemux, QtDemuxStream * stream,
     return FALSE;
   }
 
-  buf = gst_buffer_new ();
-  GST_BUFFER_DATA (buf) = tenc->data;
-  GST_BUFFER_SIZE (buf) = GST_READ_UINT32_BE (tenc->data);
-  g_signal_emit (qtdemux, gst_qtdemux_signals[SIGNAL_CENC_TENC], 0,
-      stream->track_id, buf);
-  gst_buffer_unref (buf);
-
-  return TRUE;
+  return fluc_drm_cenc_parse_tenc(stream->cenc_context, tenc->data, GST_READ_UINT32_BE (tenc->data));
```

Comment:
should be
fluc_drm_cenc_parse_tenc(stream->cenc_context, tenc->data + 12, GST_READ_UINT32_BE (tenc->data));

Reply:
because this shift is container-related, not cenc-related